presto-product-tests/src/main/java/io/prestosql/tests/cli/TestPrestoLdapCli.java

```diff
@@ -274,7 +274,7 @@ private void launchPrestoCliWithServerArgument(String... arguments)
                 "--server", ldapServerAddress,
                 "--truststore-path", ldapTruststorePath,
                 "--truststore-password", ldapTruststorePassword,
-                "--user", ldapUserName,
+                "--user", escapeForBash(ldapUserName),
```

Comment:
any reason to escape user name but not eg ldapTruststorePassword?

Reply:
ldapUserName is tested to be empty. TrustStore is always provided

Reply:
i don't understand this?